superset/views/core.py

```diff
@@ -1912,6 +1914,7 @@ def dashboard(
             bootstrap_data=json.dumps(
                 bootstrap_data, default=utils.pessimistic_json_iso_dttm_ser
             ),
+            standalone_mode=standalone_mode,
```

Comment:
nit: since `standalone_mode` isn't being reused, I'd prefer to place it inline here, like
```python
standalone_mode=ReservedUrlParameters.is_standalone_mode(),
```